### PR TITLE
Updated the readme.md by fixing the hyperlinks 

### PR DESCRIPTION
**Pull request by Ghat0tkach**
## Fixes Issue
Updated  the missing devops , blockchain and android hyperlinks in readme.md
Changed FullStack from Frontend heading in the Full stack roadmap readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,14 +6,13 @@ This repository contains the list of communities and job portals you can join an
...
 <p align="center"><img align="center" alt="Coding" src="https://media.giphy.com/media/xUySTOigOUHucl3rfW/giphy.gif"></p>
-<p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Backend-Development#readme"><B>Backend-Development</B></a></p>
-<p  align="center"><a href=""><b>Blockchain</b></a></p>
-<p  align="center"><a href=""><b>DevOps</b></a></p>
-<p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/DevRel#readme"><b>DevRel</b></a></p>
-<p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Frontend-Development#readme"><b>Frontend-Development</b></a></p>
-<p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Fullstack-Development#readme"><b>Fullstack-Development</b></a></p>
-<p  align="center"><a href=""><b>Android-Development</b></a></p>
-<p align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Open-Source#readme"><b>Blockchain</b></a></p>
+<p  align="center"><a href="Backend-Development#readme"><B>Backend-Development</B></a></p>
```
Comment (kaiwalyakoparkar) [inline README.md:9]: Please align the roadmap names to the left. It doesn't look visually aligned with the current readme design

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Comment by Ghat0tkach** [comment]
shall i change the position of gif image too? or let it be in center?

**Review by siddhant-khisty — APPROVED** [review]
LGTM
cc @kaiwalyakoparkar

**Review by kaiwalyakoparkar — APPROVED** [review]
Lgtm. Sorry for keeping you'll waiting so long

**Pull request merged**